clean-lumber.lic

```diff
@@ -5,81 +5,91 @@
 custom_require.call(%w[common common-items common-crafting common-money common-travel drinfomon])
 
 class CleanLumber
-  include DRC
-  include DRCC
-  include DRCI
-  include DRCM
-  include DRCT
 
   def initialize
     arg_definitions = [
       [
         { name: 'type', regex: /\w+/, description: 'wood type to process' },
         { name: 'source', regex: /\w+/, description: 'container' },
-        { name: 'storage', regex: /\w+/, description: 'container to put lumber in' }
+        { name: 'storage', regex: /\w+/, description: 'container to put lumber in' },
+        { name: 'table_saw', regex: /^table_saw/, optional: true, description: 'Use table saw' },
+        { name: 'stack', regex: /\d+/, optional: true, description: 'Optional stack size'}
       ]
     ]
     args = parse_args(arg_definitions)
     settings = get_settings
     @bag = settings.crafting_container
     @bag_items = settings.crafting_items_in_container
     @belt = settings.engineering_belt
+    @stack = args.stack || 45
 
-    %w[stick log branch limb deed].each { |size| process_size(args, size) }
-  end
-
-  def get_item(name)
-    get_crafting_item(name, @bag, @bag_items, @belt)
-  end
-
-  def stow_item(name)
-    stow_crafting_item(name, @bag, @belt)
+    %w[stick log branch limb deed].each { |size| 
+      loop do
+        break unless process_size(args, size)
+        cut_and_stack(args)
+      end
+      }
```

Comment:
For a multi-line `.each`, the `.each do |size|` without curly braces is preferred.